### DISCUSSION TITLE
openfast cannot run in Windows Server 2019

### DISCUSSION THREAD

**Discussion by clun1** [OpenFAST/openfast#1928] [General]
I want to run fast.farm on the Windows server 2019 operating system, but the following issues occur. However, when fast.farm is run on the Windows 10 operating system, it will work normally. I want to know if fast.farm does not work on the server?
![image](https://github.com/OpenFAST/openfast/assets/93849049/d2eddc03-7c44-4927-b1df-49e7f7a7bf4b)

**Comment by jjonkman** [comment]
Dear @clun1,

Are you using a precompiled binary executable of FAST.Farm provided by NREL or have you compiled FAST.Farm yourself?  If the former, I would suggest trying to compile FAST.Farm yourself on the Windows server.

Best regards,
